marlowe-playground-client/static/css/panels.scss

```diff
@@ -28,6 +28,10 @@
   align-self: flex-start;
 }
 
+.align-center {
```

Comment:
If we are going to use tailwind convention for the CSS utilities this could be called `self-center` 

https://tailwindcss.com/docs/align-self

Reply:
I will have it in mind in the future. For this one in particular `align-top` was already there, so it is probably best to have them consistent